### PR TITLE
Proof P30_P37

### PR DESCRIPTION
**Pull request by Lifeindeath**
Fixing issue #613

### CODE REVIEW

File: Luminary099/P30_P37.agc [hunk 1]
```diff
@@ -33,26 +33,26 @@
 #
 # MOD.1 BY RAMA AIYAWAR
 #
```
Comment (wopian) [inline Luminary099/P30_P37.agc:35]:
```suggestion
```

File: Luminary099/P30_P37.agc [hunk 2]
```diff
@@ -33,26 +33,26 @@
...
-# FUNCTIONAL DESCRIPTIONS
-#	ACCEPT ASTRONAUT INPUTS OF TIG,DELV(LV)
+# FUNCTIONAL DESCRIPTION
+#	ACCEPT ASTRONAUT INPUTS OF TIG.DELV(LV)
 #	CALL IMU STATUS CHECK ROUTINE (R02)
-#	DISPLAY TIME TO GO, APOGEE, PERIGEE, DELV(MAG), MGA AT IGN
+#	DISPLAY TIME TO GO.  APOGEE, PERIGEE, DELV(MAG), MGA AT IGN
```
Comment (wopian) [inline Luminary099/P30_P37.agc:39]: Badly printed comma
```suggestion
#	DISPLAY TIME TO GO, APOGEE, PERIGEE, DELV(MAG), MGA AT IGN
```

File: Luminary099/P30_P37.agc [hunk 3]
```diff
@@ -65,10 +65,10 @@ P30		TC	UPFLAG		# SET UPDATE FLAG
 		ADRES	TRACKFLG
 
 P30N33		CAF	V06N33		# T OF IGN
-		TC	VNP00H		# RETURN ON PROCEED, P00H ON TERMINATE
+		TC	VNPOOH		# RETURNS ON PROCEED, POOH ON TERMINATE
 
 		CAF	V06N81		# DISPLAY DELTA V (LV)
-		TC	VNP00H		#	REDISPLAY ON RECYCLE
+		TC	VNPOOH		#	REDISPLAY ON RECYCLE
```
Comment (wopian) [inline Luminary099/P30_P37.agc:71]:
```suggestion
		TC	VNPOOH		#     REDISPLAY ON RECYCLE
```

File: Luminary099/P30_P37.agc [hunk 4]
```diff
@@ -77,14 +77,14 @@ P30N33		CAF	V06N33		# T OF IGN
 			S30.1
 		SET	EXIT
 			UPDATFLG
-PARAM30		CAF	V06N42		# DISPLAY APOGEE,PERIGEE,DELTA V
-		TC	VNP00H
+PARAM30		CAF	V06N42		# DISPLAY APOGEE, PERIGEE, DELTA V
```
Comment (wopian) [inline Luminary099/P30_P37.agc:80]: image
```suggestion
PARAM30		CAF	V06N42		# DISPLAY APOGEE,PERIGEE ,DELTA V
```

File: Luminary099/P30_P37.agc [hunk 5]
```diff
@@ -77,14 +77,14 @@ P30N33		CAF	V06N33		# T OF IGN
...
+		TC	VNPOOH
 # Page 615
 
 		TC	INTPRET
 		SETGO
 			XDELVFLG	# FOR P40'S: EXTERNAL DELTA-V GUIDANCE.
-			REVN1645	# TRKMKCNT, T60, +MGA  DISPLAY
+			REVN1645	# TRKMKCNT, T60, +MGA DISPLAY
```
Comment (wopian) [inline Luminary099/P30_P37.agc:87]:
```suggestion
			REVN1645	# TRKMKCNT, T60, +MGA  DISPLAY
```

File: Luminary099/P30_P37.agc [hunk 6]
```diff
@@ -96,7 +96,7 @@ V06N42		VN	0642
 #
 # FUNCTIONAL DESCRIPTION
 #	BASED ON STORED TARGET PARAMETERS (R OF IGNITION (RTIG), V OF
```
Comment (wopian) [inline Luminary099/P30_P37.agc:98]:
```suggestion
#	BASED ON STORED TARGET PARAMETERS(R OF IGNITION(RTIG),V OF
```

File: Luminary099/P30_P37.agc [hunk 7]
```diff
@@ -96,7 +96,7 @@ V06N42		VN	0642
...
-#	IGNITION (VTIG), TIME OF IGNITION (TIG)), COMPUTE PERIGEE ALTITUDE
+#	IGNITION (VTIG), TIME OF IGNITION (TIG)).  COMPUTE PERIGEE ALTITUDE
```
Comment (wopian) [inline Luminary099/P30_P37.agc:99]:
```suggestion
#	IGNITION(VTIG),TIME OF IGNITION (TIG)),COMPUTE PERIGEE ALTITUDE
```

File: Luminary099/P30_P37.agc [hunk 8]
```diff
@@ -96,7 +96,7 @@ V06N42		VN	0642
...
 #	APOGEE ALTITUDE AND DELTAV REQUIRED (DELVSIN).
```
Comment (wopian) [inline Luminary099/P30_P37.agc:100]:
```suggestion
#	APOGEE ALTITUDE AND DELTAV REQUIRED(DELVSIN).
```

File: Luminary099/P30_P37.agc [hunk 9]
```diff
@@ -96,7 +96,7 @@ V06N42		VN	0642
 #
```
Comment (wopian) [inline Luminary099/P30_P37.agc:96]:
```suggestion
```

File: Luminary099/P30_P37.agc [hunk 10]
```diff
@@ -96,7 +96,7 @@ V06N42		VN	0642
...
 # FUNCTIONAL DESCRIPTION
 #	BASED ON STORED TARGET PARAMETERS (R OF IGNITION (RTIG), V OF
-#	IGNITION (VTIG), TIME OF IGNITION (TIG)), COMPUTE PERIGEE ALTITUDE
+#	IGNITION (VTIG), TIME OF IGNITION (TIG)).  COMPUTE PERIGEE ALTITUDE
 #	APOGEE ALTITUDE AND DELTAV REQUIRED (DELVSIN).
 #
```
Comment (wopian) [inline Luminary099/P30_P37.agc:101]:
```suggestion
```

File: Luminary099/P30_P37.agc [hunk 11]
```diff
@@ -96,7 +96,7 @@ V06N42		VN	0642
...
 # CALLING SEQUENCE
```
Comment (wopian) [inline Luminary099/P30_P37.agc:102]: Line 105, no empty line before `NORMAL EXIT MODE`

Line 108, no empty line before `SUBROUTINES CALLED`

Line 112, no empty line before `ALARM OR ABORT`

Line 115, no empty line before `ERASABLE INITIALIZATION`

File: Luminary099/P30_P37.agc [hunk 12]
```diff
@@ -117,22 +117,22 @@ V06N42		VN	0642
...
 # OUTPUT
-#	RTIG		POSITION AT TIG		VECTOR B+29 METERS
-#	VTIG		VELOCITY AT TIG		VECTOR B+29 METERS/CS
-#	PDL 4D		APOGEE ALTITUDE		DP B+29 M, B+27 METERS.
-#	HAPO		APOGEE ALTITUDE		DP B+29 METERS
-#	PDL 8D		PERIGEE ALTITUDE	DP B+29 M, B+27 METERS.
-#	HPER		PERIGEE ALTITUDE	DP B+29 METERS
+#	RTIG		POSITION AT TIG		VECTOR 	B+29 METERS
+#	VTIG		VELOCITY AT TIG		VECTOR 	B+29 METERS/CS
+#	PDL 4D		APOGEE ALTITUDE		DP 	B+29 M, B+27 METERS.
```
Comment (wopian) [inline Luminary099/P30_P37.agc:125]:
```suggestion
#	PDL 4D		APOGEE ALTITUDE		DP 	B+29 M ,  B+27 METERS.
```

File: Luminary099/P30_P37.agc [hunk 13]
```diff
@@ -117,22 +117,22 @@ V06N42		VN	0642
...
+#	HAPO		APOGEE ALTITUDE		DP 	B+29 METERS
+#	PDL 8D		PERIGEE ALTITUDE	DP 	B+29 M, B+27 METERS.
```
Comment (wopian) [inline Luminary099/P30_P37.agc:127]:
```suggestion
#	PDL 8D		PERIGEE ALTITUDE	DP 	B+29 M ,  B+27 METERS.
```

File: Luminary099/P30_P37.agc [hunk 14]
```diff
@@ -117,22 +117,22 @@ V06N42		VN	0642
...
 #			COORD. OF ACTIVE VEHICLE AT
-#			TIME OF IGNITION	VECTOR B+7 METERS/CS
-#	DELVSAB		MAG. OF DELVSIN		VECTOR B+7 METERS/CS
+#			TIME OF IGNITION	VECTOR 	B+7 METERS/CS
+#	DELVSAB		MAG. OF DELVSIN		VECTOR 	B+7 METERS/CS
 #
-# DEBRIS	QTEMP	TEMP.ERASABLE
-#		QPRET, MPAC
+# DEBRIS	QTEMP	TEMP. ERASABLE
+#		QPRET.MPAC
```
Comment (wopian) [inline Luminary099/P30_P37.agc:135]: Badly printed comma
```suggestion
#		QPRET,MPAC
```

File: Luminary099/P30_P37.agc [hunk 15]
```diff
@@ -146,7 +146,7 @@ S30.1		STQ	DLOAD
 		STCALL	TDEC1
 			LEMPREC		# ENCKE ROUTINE FOR LEM
 
-		VLOAD	SXA,2
+		VLOAD	SXA.2
```
Comment (wopian) [inline Luminary099/P30_P37.agc:149]:
```suggestion
		VLOAD	SXA,2
```

File: Luminary099/P30_P37.agc [hunk 16]
```diff
@@ -157,7 +157,7 @@ S30.1		STQ	DLOAD
 		STORE	VTIG
 		VXV	UNIT
 			RTIG
-		SETPD	SXA,1
+		SETPD	SXA.1
```
Comment (wopian) [inline Luminary099/P30_P37.agc:160]:
```suggestion
		SETPD	SXA,1
```

File: Luminary099/P30_P37.agc [hunk 17]
```diff
@@ -157,7 +157,7 @@ S30.1		STQ	DLOAD
...
 			0
 			RTX1
 		PUSH	VXV		# YRF/LV PDL 0 SCALED AT 2
```
Comment (wopian) [inline Luminary099/P30_P37.agc:163]:
```suggestion
		PUSH	VXV		# YRF/LV PDL 0  SCALED AT 2
```

Comment (wopian) [inline Luminary099/P30_P37.agc:163]: Line 166 (YRF/LV), triple space before `SCALED` instead of single

File: Luminary099/P30_P37.agc [hunk 18]
```diff
@@ -187,7 +187,7 @@ S30.1		STQ	DLOAD
 			SHIFTR1		# RESCALE IF NEEDED
```
Comment (wopian) [inline Luminary099/P30_P37.agc:187]: Line 184 (`PERIGEE ALT`), comma after `METERS`

File: Luminary099/P30_P37.agc [hunk 19]
```diff
@@ -187,7 +187,7 @@ S30.1		STQ	DLOAD
...
 		CALL			# LIMIT DISPLAY TO 9999.9 N. MI.
 			MAXCHK
-		STCALL	HAPO		# APOGEE ALT 2(29) METERS FOR DISPLAY
+		STCALL	HAPO		# APOGEE ALT 2(29) METERS, FOR DISPLAY
```
Comment (wopian) [inline Luminary099/P30_P37.agc:190]:
```suggestion
		STCALL	HAPO		# APOGEE  ALT 2(29) METERS, FOR DISPLAY
```

**Review by wopian — CHANGES_REQUESTED** [review]
Thank you for opening this PR. I've added some code suggestions for missed typos (or accidental replacement of correct characters - mainly badly printed commas that plague these scans).

Two of these are out of range of the code suggestions feature and will need to be committed manually by you.

- https://github.com/chrislgarry/Apollo-11/pull/709/files#r502030829
- https://github.com/chrislgarry/Apollo-11/pull/709/files#r502033150

**Comment by wopian** [comment]
Hey @Lifeindeath, this PR is waiting on 2 changes I couldn't commit for you that are linked in https://github.com/chrislgarry/Apollo-11/pull/709#pullrequestreview-505184446 (or above).

Once those changes have been committed, this is good to be merged 👍

**Comment by chrislgarry** [comment]
@Lifeindeath if you're still up for it, this can be merged once you commit the changes @wopian shared in the comments. Thanks!

**Review by wopian — APPROVED** [review]

**Comment by wopian** [comment]
Thank you @Lifeindeath for proofing this file 👍

**Pull request merged**